src/Symfony/Component/Cache/Adapter/FilesystemAdapter.php

```diff
@@ -20,10 +20,16 @@ class FilesystemAdapter extends AbstractAdapter
 {
     private $directory;
 
-    public function __construct($directory, $defaultLifetime = null)
+    public function __construct($directory, $defaultLifetime = 0, $namespace = '')
     {
         parent::__construct('', $defaultLifetime);
```

Comment:
no because we handle it differently here: we create a subdir to isolate "namespaces" so there is no need to isolate using key prefixing by the abstract adapter